File: MODUL 4 PEMROGRAMAN 1/PRAK403-2310817120002-SITI RATNA DWINTA SARI.py
for u in range(4):
    bil1, bil2 = map(int, input().split())

    for i in range(bil1, bil2 + 1):
        print(f"{i} {bil2 - (i - bil1)}", end="")
        if i < bil2:
            print(" - ", end="")
    
    for i in range(bil1, bil2 - 1, -1):
        print(f"{i} {bil2 - (i - bil1)}", end="")
        if i > bil2:
            print(" - ", end="")
    
    print()

File: MODUL 4 PEMROGRAMAN 1/PRAK403-2310817120002-SITIRATNADWINTASARI.c
#include <stdio.h>

int main() {
    int u, i, bil1, bil2;
    for(u = 1; u <= 4; u++){
    scanf("%d %d", &bil1, &bil2);

    for (i = bil1; i <= bil2; i++) {
        printf("%d %d", i, bil2 - (i - bil1));
        if (i < bil2) {
            printf(" - ");
     }
        }  
        for (i = bil1; i >= bil2; i--){
            printf("%d %d", i, bil2 - (i - bil1));
            if (i > bil2) {
                printf(" - ");
            }
        }
        printf("\n");
    }
     }